frontend/src/metabase/admin/databases/containers/DatabaseEditApp.unit.spec.js

```diff
@@ -41,7 +41,7 @@ const ENGINES_MOCK = {
 
 const ComponentMock = () => <div />;
 jest.mock(
-  "metabase/databases/containers/DatabaseHelpCard",
+  "metabase/databases/components/DatabaseHelpCard",
```

Comment:
If we want to keep things clean, I can remove this change.
This is a leftover from https://github.com/metabase/metabase/pull/32367/commits/3eaf88584ccdfa51a5d2fce78abefb48afff6abb

cc @ranquild